src/Tools.pm

```diff
@@ -401,31 +412,41 @@ DMRaid Devices look like:
 DMRaid Partitions look like:
 <strange name>_part\d
 
-
 =cut
 
-sub ReadDMRaidPartitions {
+# FIXME: this has to be done through Storage
```

Comment:
this FIXME is wrong, as Tools module is intended to be used during kernel update and its goal is to be completelly independent on yast and also it should not use storage.